lib/code_corps_web/controllers/organization_github_app_installation_controller.ex

```diff
@@ -1,18 +1,12 @@
 defmodule CodeCorpsWeb.OrganizationGithubAppInstallationController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
   import CodeCorps.Helpers.Query, only: [id_filter: 2]
 
-  alias CodeCorps.{OrganizationGithubAppInstallation}
+  alias CodeCorps.{OrganizationGithubAppInstallation, User, Helpers.Query}
 
-  @preloads [:github_app_installation, :organization]
-
-  plug :load_resource, model: OrganizationGithubAppInstallation, only: [:show], preload: @preloads
-  plug :load_and_authorize_changeset, model: OrganizationGithubAppInstallation, only: [:create], preload: @preloads
-  plug :load_and_authorize_resource, model: OrganizationGithubAppInstallation, only: [:delete]
-
-  plug JaResource
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
 
   @spec model :: module
   def model, do: CodeCorps.OrganizationGithubAppInstallation
```

Comment:
Sorry I missed this on the first pass, but the method here is no longer necessary now, so you can remove it.

So I don't annoy you further, don't forget to squash again, as well as run a `git pull develop` followed by `git rebase origin/develop` at the end of it. Then I can merge.

Reply:
ok thanks for your patience @begedin with my first contribution, I rebased on the latest origin/develop and removed `def model` method....thanks!

Reply:
ok thanks for your patience @begedin with my first contribution, I rebased on the latest origin/develop and removed def model method....thanks!

Reply:
I'm the one who should be thankful @treble37 

And now I have to apologize again and ask you to do yet another change. The `filter` method also isn't required anymore. Mind removing that one to? 

Again, squash and rebase after.

I'm really sorry about this, but the "changes" view on github really sucks when it comes to seeing which code needs to be removed.

Reply:
No worries @begedin, filter removed!